Gruntfile.js

```diff
@@ -48,9 +48,7 @@ module.exports = function(grunt) {
           'test/lib/unittests-bundle.js': [ './test/unittests.js' ]
         },
         options: {
-          browserifyOptions: {
-            external: [ 'openpgp', 'crypto', 'node-localstorage']
-          }
+          external: [ 'openpgp']
```

Comment:
We should keep 'crypto' 'node-localstorage' external for the browser unit tests. Any reason you added them here?